.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: trusty
 jdk:
 - openjdk8
 install: true # skips travis' default installation step which executes gradle assemble.
-script: ./gradlew clean build --daemon
```

Comment:
Can you explain why this change?

Reply:
Sorry by mistake.
I recovered it.
https://github.com/commercetools/commercetools-project-sync/pull/147/commits/072b19251f6d98a8dd3317b4f26f5d21fa1ba1c2